app/Recording.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class Recording
+ * @package App
+ */
+class Recording extends Model
+{
+    /**
+     * @string
+     */
+    private $title;
+
+    /**
+     * @string
+     */
+    private $artist;
```

Comment:
Instead of using @string type, you should set up a OneToOne relationship to Artist model 
https://laravel.com/docs/8.x/eloquent-relationships